sagemaker-inference-recommender/deployment-recommendations/instant-recommendations.ipynb

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
<p>Where does the model come from? Is it an empty model? If not, does it have legal approval?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4077/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>hey yeah its just an empty model that takes in a sagemaker dlc. thats it</p>